x-pack/plugins/fleet/common/openapi/bundled.yaml

```diff
@@ -182,7 +180,9 @@ paths:
                       id:
                         type: string
                         nullable: true
-                        description: the key ID of the GPG key used to verify package signatures
+                        description: >-
```

Comment:
these changes were generated by the `@redocly/openapi-cli` command, not sure where the extra whitespace is coming from

Reply:
It's possible the `redocly` dependency was updated since the last time we built the OpenAPI spec and so this is just some upstream change 🤷